recipes/semchunk/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "semchunk" %}
+{% set version = "2.2.0" %}
+{% set build_number = 0 %}
```

Comment:
It's more typical to just put the build number directly under `build` rather than use a jinja variable